src/main/ks-installer/Chart.yaml

```diff
@@ -18,9 +18,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.2
+version: 1.0.0
```

Comment:
I think it might be better to change the chart version to `0.3.0`.

Reply:
But this update has changed the Values structure which is incompatible with old version.

Reply:
Done.